src/main/java/bio/terra/service/filedata/google/firestore/FireStoreFile.java

```diff
@@ -198,7 +198,7 @@ public int hashCode() {
   public static FireStoreFile fromTableEntity(TableEntity entity) {
     return new FireStoreFile()
         .fileId(entity.getProperty(FILE_ID_FIELD_NAME).toString())
-        .mimeType(entity.getProperty(MIME_TYPE_FIELD_NAME).toString())
+        .mimeType((String) entity.getProperty(MIME_TYPE_FIELD_NAME))
```

Comment:
Note that this is different if `entity.getProperty(MIME_TYPE_FIELD_NAME)` returns something other than `String`.

Also I see `toString()` is used on other fields here. Is it specified as part of the azure APIs which fields can and can't be `null`?

Reply:
I think the other fields should always be defined, but I can change those so that they are cast to `String` too.

Reply:
Also the mimetype comes from `fileLoadModel.getMimeType()` so it will always be a string!